roles/gateway_processing_node/tasks/main.yml

```diff
@@ -10,19 +10,15 @@
 - name: Retrieve node properties template from source gateway, update properties, and use it to auto-provision destination Gateway.
```

Comment:
retrieve from source gateway still or from property file

Reply:
Updated task name.

Reply:
Do you need to update readme to mention prerequisite of the role is populating the property  template file?